test/gtest/ucp/ucp_test.h

```diff
@@ -211,6 +211,8 @@ class ucp_test : public ucp_test_base,
     void stats_activate();
     void stats_restore();
 
+    void disable_keepalive();
```

Comment:
do we really need it in base ucp_test class?

Reply:
why not?

Reply:
just because it's not used anywhere else excepting new test